### PR TITLE
Add the version flag to the fybrik helm installation

### LINKED ISSUE #923: Quickstart install documentation is not version specific, but rather install from latest release
Fybrik quickstart documentation contains sections on
1. Install control plane
2. Install modules
which install the latest release instead of the specific release the quick-start web page refers to, e.g., v0.3, v0.4, ...

### LINKED ISSUE #1169: Wrong fybrik installallation commands
The [quickstart ](https://fybrik.io/v0.5/get-started/quickstart/#install-control-plane) for version 0.5.x recommends to run:
```
helm install fybrik-crd fybrik-charts/fybrik-crd -n fybrik-system --wait
helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait
```
but it will install the `latest` fybrik version - v0.6.0

### PR DESCRIPTION
**Pull request by roytman**
Adds the version flag to the fybrik helm installation
/fixes #923 
/fixes #1169

Signed-off-by: Alexey Roytman <roytman@il.ibm.com>

### CODE REVIEW

**Review by revit13 — APPROVED** [review]

File: site/docs/get-started/quickstart.md [hunk 1]
```diff
@@ -128,10 +142,10 @@ helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait
 
 [Modules](../concepts/modules.md) are plugins that the control plane deploys whenever required. The [arrow flight module](https://github.com/fybrik/arrow-flight-module) enables reading data through Apache Arrow Flight API. 
 
-Install the latest[^1] release of arrow-flight-module:
+Install the {{ arrowFlightRelease }}[^1] release of arrow-flight-module:
 
 ```bash
-kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/latest/download/module.yaml -n fybrik-system
+kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/download/{{ arrowFlightRelease }}/module.yaml -n fybrik-system
```
Comment (roee88) [inline site/docs/get-started/quickstart.md:148]: Note that https://github.com/fybrik/arrow-flight-module/releases/download/latest/module.yaml is not a valid link

Comment (roytman) [reply site/docs/get-started/quickstart.md:148]: Thanks, will fix it to `https://github.com/fybrik/arrow-flight-module/releases/latest/download/module.yaml`

Comment (roytman) [reply site/docs/get-started/quickstart.md]: fixed

**Comment by roytman** [comment]
@roee88 , any additional comments?

File: RELEASE.md [hunk 2]
```diff
@@ -25,6 +25,24 @@ Collaborators with `write` permissions can cherry-pick a Pull Request that is me
 
 You should ensure that all Pull Requests that target the release branch are reviewed and merged before proceeding to the next step.
 
+## 3. Optionally update the ArrowFlight releases file
+
+The file `./site/arrowFlightReleases` contains a map between fybrik and teh arrow-flight-module releases.
```
Comment (revit13) [inline RELEASE.md:30]: teh -> the

Comment (roytman) [reply RELEASE.md]: fixed, thanks

**Pull request merged**